### ISSUE TITLE
Are advertising=billboard (with name tag) searchable on Nominatim?

### ISSUE THREAD

**Issue by ReinoBaptista** [osm-search/Nominatim#2188]
## What did you search for?

"Painel da Pan Am"

## What result did you get?

Nothing

## What result did you expect?

"https://www.openstreetmap.org/node/7561154168"

The result misses completely

## Further details

When/If I add tourism=attraction to the node there are results.

**Comment by lonvia** [comment]
Object with `advertising=*` are not imported at the moment. I don' t see that happen anytime soon. They are not the usual thing people search for and very rarely have a name to search by anyway.

**Comment by ReinoBaptista** [comment]
Hi,

Ok. I accept that. But...
Shouldn't every object with a name be imported? If people mapped it with a name wouldn't it be interesting per se?

More importante:
Is there a list somewhere of what is imported (or not imported)?
Thank you very much.

**Comment by lonvia** [comment]
> Shouldn't every object with a name be imported? If people mapped it with a name wouldn't it be interesting per se?

Yes and no. The problem with the 'name' tag in OSM is that it tends to get (ab)used to hold a description instead of the name. In its current state Nominatim isn't very well suited to work with that kind of names. So it restricts the imported objects a bit. I hope we can change this in the future and then we can revisit billboards as well. That's why I haven't closed this issue yet.

> Is there a list somewhere of what is imported (or not imported)?

The configuration file is here: https://github.com/osm-search/Nominatim/blob/master/settings/import-extratags.style
The documentation has [some explanations](https://nominatim.org/release-docs/latest/develop/Import/#configuring-the-import) about the format.

**Comment by ReinoBaptista** [comment]
Thank you Sarah
